website/docs/intl.md

```diff
@@ -74,6 +74,7 @@ Creates a cache instance to be used globally across locales. This memoizes previ
 interface IntlConfig {
   locale: string
   timeZone?: string
+  fallbackOnEmptyString?: boolean
```

Comment:
@isaachinman mind adding some docs that explain that options down in the section right below `defaultRichTextElements`? you can link the GH ticket w/ the discussion as well

Reply:
Yep, added a brief section. Let me know if you think it's enough.